docs/en-US/component/container.md

```diff
@@ -79,56 +79,66 @@ container/example
 
 :::
 
-## Container Attributes
+## Container API
 
-| Name      | Description                         | Type   | Accepted Values       | Default                                                                    |
-| --------- | ----------------------------------- | ------ | --------------------- | -------------------------------------------------------------------------- |
-| direction | layout direction for child elements | string | horizontal / vertical | vertical when nested with `el-header` or `el-footer`; horizontal otherwise |
+### Container Attributes
 
-## Container Slots
+| Name      | Description                         | Type                                  | Default                                                                    |
+| --------- | ----------------------------------- | ------------------------------------- | -------------------------------------------------------------------------- |
+| direction | layout direction for child elements | ^[string]`'horizontal' \| 'vertical'` | vertical when nested with `el-header` or `el-footer`; horizontal otherwise |
 
-| Name | Description               | Subtags                                    |
-| ---- | ------------------------- | ------------------------------------------ |
-| —    | customize default content | Container / Header / Aside / Main / Footer |
+### Container Slots
 
-## Header Attributes
+| Name    | Description               | Subtags                                                      |
+| ------- | ------------------------- | ------------------------------------------------------------ |
+| default | customize default content | ^[Component]`Container \| Header \| Aside \| Main \| Footer` |
```

Comment:
```suggestion
| default | customize default content | Container / Header / Aside / Main / Footer |
```

It might be better to keep the previous value, what you think?

Reply:
Regarding this point, I am also not sure whether it will be better after modification, and I will restore it.